RubeesEat.IntegrationTests/SeleniumTests/RubeesEatSeleniumTests.cs

```diff
@@ -0,0 +1,21 @@
+using RubeesEat.IntegrationTests.PageObjects;
+
+namespace RubeesEat.IntegrationTests.SeleniumTests;
+
+[TestFixture]
+public class RubeesEatSeleniumTests : SeleniumIntegrationTestBase
+{
+    public RubeesEatSeleniumTests() : base("UserHomepage")
+    {
+    }
+    
+    [Test]
+    public void GetDefaultBalanceChanges()
+    {
+        var page = Start<UserHomepagePageObject>();
+
+        Assert.That(
+            page.GetBalanceChangeTexts(),
+            Is.EqualTo(new[] { "1000€ Tuesday 19.08.2025 Default user paid lunch 2025", "-800€ Sunday 28.04.2024 Lunch with default user in 2024", "-800€ Friday 28.04.2023 Lunch with default user in 2023", "-800€ Thursday 28.04.2022 Lunch with default user 2022" }));
```

Comment:
Ideally we don't just assert the text but also the structure. That requires us to render the elements (amount, description, date) in separate HTML elements that we can select. We can then retrieve these fields and construct a `BalanceChange` object in the page object. This allows us to do this assert on objects and no longer assert text here.